bricksrc/status.py

```diff
@@ -4,6 +4,9 @@
     "Status": {
         "tags": [TAG.Point, TAG.Status],
         "subclasses": {
+            "Damper_Position_Status": {
+                BRICK.hasQuantity: BRICK.Position,
```

Comment:
Shouldn't this be `Damper_Position`?

Reply:
Do we want equipment-flavored quantities? I don't believe we have any others, e.g. "fan speed"